_includes/banner.html

```diff
@@ -10,6 +10,10 @@
         <span>&nbsp;&nbsp;</span>
         <a href="/documentation/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-book"></i> Documentation</button></a>
         <span>&nbsp;&nbsp;</span>
+        <a href="/resources/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-external-link-square"></i> Resources</button></a>
+        <span>&nbsp;&nbsp;</span>
+        <a href="/resources/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-rss-square"></i> Ehcache Blog</button></a>
```

Comment:
Actually I agree with you but I don't know the answer because when discussing with @ljacomet we decided to put resources there. I sincerely don't know now if it is better to move it or change the CSS to improve the font / space or anything else.

The issue currently is that the top bar is too long and on smaller resolution, the right side comes below the left side and it's ugly.